db/seeds.rb

```diff
@@ -50,3 +50,13 @@
     parent_id: comment.id
   )
 end
+# Generate votes for articles
+articles.each do |article|
+  rand(1..5).times do  # Random number of votes per article
+    Vote.create!(
+      votable: article,
+      user: users.sample,
```

Comment:
Once the unique validations are in place, this could cause some trouble due to the randomness of it.